src/applications/MachineShopSimulator.java

```diff
@@ -183,6 +175,20 @@ public  SimulationResults runSimulation(SimulationSpecification specification) {
     public Machine getMachine(int p) {
         return machine[p];
     }
+    /**
+     * get the current time.
+     * @return timeNow
+     */
+    public int getTimeNow(){
+        return timeNow;
+    }
+    /**
+     * get the large time from the MSS class.
+     * @return
+     */
+    public int getLargeTime(){
+        return largeTime;

```

Comment:
Usually, getters are used for getting internal fields that shouldn't be publicly exposed.  I think constants probably _should_  just be publicly exposed rather than using a getter; largeTime in particular isn't really a piece of information tied to an instance of a MachineShopSimulator, simply a big number.

Reply:
This makes sense, I guess I didn't really understand how they were being used.